### ISSUE TITLE
Tensor size mismatch on M3ED dataset

### ISSUE THREAD

**Issue by ansucatherine** [chengzju/CARAT#7]
Dear authors,

Thank you for open-sourcing your code. While running it on the M3ED dataset, I encountered a RuntimeErrorin loss.py(line 37):
```shell
The size of tensor a (11648) must match the size of tensor b (12224) at non-singleton dimension 1
```
The error occurs during the operation `exp_logits = torch.exp(logits) * logit_mask`, likely due to a dimension mismatch between logits(($111648\times11648$)) and logit_mask($11648\times12224$).

Upon debugging, I noticed that such an issue will be alleviated when the condition `cl_labels.shape[0] = batch_size` holds, implying `bs * 3c * label_time + queue_size = bs * 6 * 9 + queue_size`. This works for CMU-MOSEI (where `c=6` and `label_time=3`), but for M3ED (with 7 classes), I couldn’t find a suitable `label_time` to satisfy the equation, leading to the dimension error.

Could you kindly share more detailed configuration guidelines for the M3ED dataset? I really appreciate your time and effort. 

Best regards

**Comment by Tianba0116** [comment]
您好，请问这个 M3ED 数据集 方便提供一下吗？我在处理的时候 找不到 train_valid_test.pt 文件
